strapi/strapi
# Spelling

### What does it do?

This PR corrects misspellings identified by the [check-spelling action](https://github.com/marketplace/actions/check-spelling).

### Why is it needed?

The misspellings have been reported at https://github.com/jsoref/strapi/commit/8f37571a31b7fd3d0c1f733c39074addbd88923d#commitcomment-69624803

### How to test it?

The action reports that the changes in this PR would make it happy: https://github.com/jsoref/strapi/commit/d7608ed558df5d065e6bfcbfd982f54c306ba9aa

Note: this PR does not include the action. If you're interested in running a spell check on every PR and push, that can be offered separately.

### Related issue(s)/PR(s)

Let us know if this is related to any issue/pull request
